### PR TITLE
onefetch: update to 2.22.0

### PR DESCRIPTION
**Pull request by voidisnull**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)
- I built this PR locally for these architectures:
  - i686
  - x86_64-glibc

### CODE REVIEW

**Comment by tranzystorekk** [comment]
Could you add manpage installation to `post_install`? It's in `docs/onefetch.1`

**Comment by voidisnull** [comment]
I'm so sorry for the late reply!
I'm actually pretty new to all this stuff so could you please guide me a little bit ?

**Comment by tranzystorekk** [comment]
For a general overview search for `vman`, `vtargetrun`, `vcompletion` in Manual.md

A good example should be the [ripgrep template](https://github.com/void-linux/void-packages/blob/3d8aa404113d6a6e3684b514dbe8fe6951463699/srcpkgs/ripgrep/template)

**Comment by voidisnull** [comment]
I'm once again sorry for the delay (I was busy with all my assignments and stuff)

@tranzystorekk 
I have added the manpages and completions
Could you please cross-check it??

File: srcpkgs/onefetch/template [hunk 1]
```diff
@@ -26,4 +27,13 @@ pre_build() {
...
 post_install() {
 	vlicense LICENSE.md
+
+	local _onefetch=${DESTDIR}/usr/bin/onefetch
+
+	vman docs/onefetch.1
+
+	for _shell in bash fish zsh; do
+		vtargetrun ${_onefetch} --generate ${_shell} > onefetch.${_shell}
+		vcompletion onefetch.${_shell} ${_shell} onefetch
```
Comment (tranzystorekk) [inline srcpkgs/onefetch/template:37]:
```suggestion
		vcompletion onefetch.${_shell} ${_shell}
```

if the binary has the same name as the package, you don't need to specify it explicitly

Comment (voidisnull) [reply srcpkgs/onefetch/template]: Thanks for you help!!

**Pull request merged**